apps/landing/components/changelog/side-list.tsx

```diff
@@ -14,15 +15,16 @@ export function SideList({ logs, className }: SideListProps) {
       <ScrollArea className="h-96 changelog-gradient xl:sticky">
         {logs?.map((changelog, _index) => {
           return (
-            <a
+            <Link
```

Comment:
Does scroll still work with a Link ? Prior it did nothing

Reply:
Works OK now